src/go/rpk/pkg/config/schema.go

```diff
@@ -42,6 +43,12 @@ func (c *Config) File() *Config {
 	return c.file
 }
 
+// RawFile returns the bytes of the actual file on disk, if there was one.
```

Comment:
I think this method can actually be removed for now, since it is not used externally?

Reply:
I use it here: https://github.com/redpanda-data/redpanda/pull/8736/files#diff-ac6177d5daadb272063ee0329c3bea5bf6e44f4b0d2c21a7d39ac833accd8031R154

Reply:
sgtm, approved 👍